### PR TITLE
Homework week 1

### PR DESCRIPTION
**Pull request by ghost**
Hi! Here's first week's homework.

### CODE REVIEW

File: homework-midy/dogPhotoGallery/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      document.getElementById('dog-list').appendChild(imageListItem);
+    }
+  };
+  xhr.open('GET', url);
+  xhr.send();
+}
+
+document.getElementById('xml').addEventListener('click', () => {
+  xmlRequest();
+});
```
Comment (nikoskleidis) [inline homework-midy/dogPhotoGallery/index.js]: No need to call a function with no arguments that calls a function with no arguments
```suggestion
document.getElementById('xml').addEventListener('click', xmlRequest);
```

File: homework-midy/programmerHumor/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+// wasn't working for some reason
+
+// ------------------------XMLHttpRequest---------------------
+const xhr = new XMLHttpRequest();
+xhr.onreadystatechange = () => {
+  if (xhr.readyState === XMLHttpRequest.DONE) {
+    const image = document.createElement('img');
+    image.setAttribute('src', JSON.parse(xhr.response).img);
+    document.body.appendChild(image);
+    console.log(JSON.parse(xhr.response, null, '\t'));
```
Comment (nikoskleidis) [inline homework-midy/programmerHumor/index.js]: console.logs added for debugging purposes should be removed before commiting

**Review by nikoskleidis — COMMENTED** [review]
Very good @MidyanHamdoun! 🥇

**Comment by ghost** [comment]
Thank you!!

**Pull request merged**